setup.py

```diff
@@ -25,7 +25,8 @@
 # Dependencies
 extras_require = {
     'serial':   ['pyserial~=3.0'],
-    'neovi':    ['python-ics>=2.12']
+    'neovi':    ['python-ics>=2.12'],
+    'mf4':      ['asammdf>=5.5.0']
```

Comment:
We also need to add `numpy` here.

```suggestion
    'mf4':      ['asammdf>=5.5.0', 'numpy>=WHATEVER_VERSION']
```